### PR TITLE
feat(api) : update CMDB code in the POD

### PR DESCRIPTION
**Pull request by soumyadip007**
Subject: feat(api) : update CMDB code in the POD
Assignees: @soumyadip007

## Closes / Fixes

1. CMDB code added for the Pod Update
2. InitEnv changed into ConfigureEnv
3. Build function for env creation zip separated
4. Rebuild added for the updation of the CMDB Code
5. Only changed CMDB code will be updated in all env of the property in every cluster

## Does this PR introduce a breaking change

NO

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api/src/server/environment/service/factory.ts [hunk 1]
```diff
@@ -36,40 +36,63 @@ export class EnvironmentFactory {
...
+    fileOriginalName: string,
+    zipPath: string,
+    propertyRequest: Property,
+    environmentRequest: Environment,
+    rebuild: boolean = false
+  ) {
+    try {
+      await this.applicationService.deployApplication(
+        buildEnvironment,
+        fileOriginalName,
```
Comment (riginoommen) [inline packages/api/src/server/environment/service/factory.ts:53]:
```suggestion
        orginalFIleName,
```

File: packages/api/src/server/environment/service/index.ts [hunk 2]
```diff
@@ -92,7 +92,8 @@ export class EnvironmentService {
     const environment = this.environmentFactory.createNewEnvironment(createEnvironmentDto);
     Promise.all([this.dataServices.environment.create(environment)]);
     // @internal TODO : removing the auto initialization, to be discussed on it further
-    // await this.environmentFactory.initializeEnvironment(property, environment);
+    // const { buildEnvironment, fileOriginalName, zipPath } = await this.environmentFactory.getArchivePath(property, environment);
+    // await this.environmentFactory.configureEnvironment(buildEnvironment, fileOriginalName, zipPath, property, environment);
```
Comment (riginoommen) [inline packages/api/src/server/environment/service/index.ts:96]:
```suggestion
```

File: packages/api/src/server/environment/service/factory.ts (hunk 1 above)
Comment (soumyadip007) [reply packages/api/src/server/environment/service/factory.ts]: Hi @riginoommen , I've changed it to `originalFileName`, please check

File: packages/api/src/server/environment/service/index.ts (hunk 2 above)
Comment (soumyadip007) [reply packages/api/src/server/environment/service/index.ts]: Removed

**Review by nikhitamore — APPROVED** [review]
I see changes have been made, rest looks good to me

**Comment by soumyadip007** [comment]
Hi @riginoommen, could you please check the changes?

**Review by riginoommen — APPROVED** [review]

**Pull request merged**